src/store/getters/state-getters.ts

```diff
@@ -522,12 +527,20 @@ export const isFilingValid = (state: StateIF): boolean => {
     // detail
     // certify (client error correction only)
     // staff payment
-    return (
-      state.stateModel.peopleAndRoles.valid &&
-      state.stateModel.detail.valid &&
-      state.stateModel.certifyState.valid &&
-      state.stateModel.staffPaymentStep.valid
-    )
+    if (state.stateModel.tombstone.correctionType === CorrectionErrorTypes.CLIENT) {
```

Comment:
when the correction was not a client correction the certifyState was not rendered and thus would never become valid